config/localization/en/app.json

```diff
@@ -1844,6 +1844,8 @@
       "CLOSE_MARKET_POSITIONS_BODY": "{MARKET} is set to close only mode and the close price of {MARKET} is now fixed. Please close your open {MARKET} position as soon as you are able. We're here to help via the help chat if you run into issues.",
       "COMPLIANCE_ALERT": "Compliance alert",
       "COMPLIANCE_WARNING": "Compliance Warning",
+      "CONNECTION_ISSUE_DETECTED": "Connection issue detected",
+      "CONNECTION_ISSUE_DETECTED_BODY": "We're having trouble connecting to our services right now. This could be due to our system status or your internet connection. Please check the status page and refresh. Your funds are safe.",
```

Comment:
due to our system status -> due to the status of our system

iono. i think that sounds a tad better